<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AreaSettingsDialog</class>
 <widget class="QDialog" name="AreaSettingsDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>168</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Ustaw obszar konkursu</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_Points">
     <property name="title">
      <string>Punkty</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_p1">
        <item>
         <widget class="QLineEdit" name="lineEdit_p1"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_p1">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/MapSelect16.png</normaloff>:/MapSelect16.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_p2">
        <item>
         <widget class="QLineEdit" name="lineEdit_p2"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_p2">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/MapSelect16.png</normaloff>:/MapSelect16.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_p3">
        <item>
         <widget class="QLineEdit" name="lineEdit_p3"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_p3">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/MapSelect16.png</normaloff>:/MapSelect16.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_p4">
        <item>
         <widget class="QLineEdit" name="lineEdit_p4"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_p4">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/MapSelect16.png</normaloff>:/MapSelect16.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_buttons">
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_restore">
       <property name="text">
        <string>Przywróć domyślne</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_buttons">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
